sound/hybrid_font.h

```diff
@@ -363,19 +383,29 @@ class HybridFont : public SaberBase {
   }
   
   bool swinging_ = false;
+  bool slashing_ = false;
   void SB_Motion(const Vec3& gyro, bool clear) override {
     float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
-    if (speed > config_.ProffieOSSwingSpeedThreshold) {
-      if (!swinging_ && state_ != STATE_OFF &&
-	  !(lockup.files_found() && SaberBase::Lockup())) {
-        swinging_ = true;
+    if (state_ != STATE_OFF && !(lockup.files_found()
```

Comment:
I wonder if we can break out the code that calls startswing/startslash into a a single function which is called from both hybrid and smoothswing fonts?

Reply:
I think we can.  We'll just need to pass in a variable to tell the new generic swing function which swing sound it will be playing.  Do you recommend a enumerated state list, so if STATE_SLASHING is set in domotion, then we can have the case statement in generic swing.

Reply:
I was thinking more along the line of just passing in the speed or the gyro vector and letting the function decide what to do.

Reply:
I'm just passing the acceleration variable into the start swing function.